packages/nx/src/command-line/init/implementation/utils.ts

```diff
@@ -25,7 +25,9 @@ export function createNxJsonFile(
   scriptOutputs: { [name: string]: string }
 ) {
   const nxJsonPath = joinPathFragments(repoRoot, 'nx.json');
-  let nxJson = {} as Partial<NxJsonConfiguration>;
+  let nxJson = {
+    $schema: './node_modules/nx/schemas/nx-schema.json',
+  } as Partial<NxJsonConfiguration> & { $schema: string };
   try {
     nxJson = readJsonFile(nxJsonPath);
```

Comment:
If we have already a nx.json file without the schema, wouldn't this override your schema setting further up?